readme.adoc

```diff
@@ -10,3 +10,38 @@ Furthermore, it provides _++include::[]++_ directives for embedding the results
 
 For more information on jQAssistant see https://jqassistant.org[^].
 
+== Using the jqassistant-asciidoc-report-plugin
+
+NOTE:: Starting from jqassistant-rdbms-plugin v2.0.0, the AsciiDoc-Report-Plugin resides under jqassistant-plugin.
```

Comment:
`Note:` oder `Note::` ist hier die Frage. Wir sollten das nur einheitlich verwenden. Das it jetzt aber nur optional und eine Anmerkung.

Reply:
👍 Ist bereits bei jedem Plugin, was ich vorbereitet habe, so.